Objects/unicodeobject.c

```diff
@@ -14657,12 +14649,14 @@ PyUnicode_InternInPlace(PyObject **p)
         return;
     }
 
+    PyObject *interned = INTERNED_DICT();
     if (interned == NULL) {
         interned = PyDict_New();
         if (interned == NULL) {
             PyErr_Clear(); /* Don't leave an exception */
             return;
         }
+        INTERNED_DICT() = interned;
```

Comment:
We've been working hard to avoid macro-as-lval in the codebase.  I recommend the following alternative:

```suggestion
        SET_INTERNED_DICT(interned);
```

Reply:
Changes to static inline functions.